src/lib/features/onboarding/onboarding-read-model.ts

```diff
@@ -86,6 +86,7 @@ export class OnboardingReadModel implements IOnboardingReadModel {
         const feature = await this.db('features')
             .select('name')
             .where('project', projectId)
+            .whereNot('archived_at', null)
```

Comment:
@kwasniew oversight, when first flag gets archived, it was returned to UI, and SDK connect dialog did not open up.